sdk/daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV2.scala

```diff
@@ -240,6 +249,21 @@ private[archive] class DecodeV2(minor: LV.Minor) {
       Work.run(decodeDefInterface(id, lfInterface))
     }
 
+    private[archive] def decodeInternedKind(lfKind: PLF.Kind): Kind = {
```

Comment:
I am a bit uneasy with this name. 
I understand this is the decoder used to decode the kind in the interning table, but on the other hand we crash 
if the message `intnerned_kind` is set.